Readme.md

```diff
@@ -90,8 +89,8 @@ We (try to) stick to the Haiku style for code, although with a few differences.
 
 Currently there are various branches on the Github repository:
```

Comment:
I'm for removing this part, branches are usually merged into the main and then deleted..